Ecosystem Partners/Marketplace Guides/getting-started-with-magento-blueprint.md

```diff
@@ -1,18 +1,18 @@
 {{{
   "title": "Getting Started with Magento - Blueprint",
-  "date": "2-25-2015",
+  "date": "02-25-2015",
   "author": "Bob Stolzberg",
   "attachments": [],
   "contentIsHTML": false
 }}}
 
-![logo](https://wiki.magento.com/download/attachments/2952491/MAGE2DOC?version=1&modificationDate=1378339878000&api=v2)
+![Magento Logo](../../images/magento-logo.png)
```

Comment:
change to `../../images/magento-appfog.png` or change the name of the file `magento-appfog.png` to `magento-logo.png'